src/models/Config.php

```diff
@@ -178,7 +178,13 @@ public function create(): bool
             ('password_complexity_requirement', '0'),
             ('max_password_age_days', '3650'),
             ('remote_dir_service', 'eairef'),
-            ('remote_dir_config', NULL)";
+            ('remote_dir_config', NULL),
+            ('onboarding_email_active', 0),
```

Comment:
everything is a string in here, so the value is `'0'`

Reply:
yeah, not a big deal I guess, it's just more *correct* to give it a string as it is stored as a string, it's fine to leave it like it is, do what you will